bot/admin/admin.py

```diff
@@ -0,0 +1,260 @@
+                        by a '|' character. If this character is not found, no title will be assumed.
+        """
+        if '|' in text:
+            title, description = text.split('|')
+        else:
+            title = ''
+            description = text
+
+        embed = discord.Embed(title=title, description=description, color=color)
+        await channel.send(embed=embed)
+
+    @commands.command(name='cembed', hidden=True)
+    @command_log
+    async def cembed(self, ctx: commands.Context, channel: discord.TextChannel, *, json_string: str):
+        """Command Handler for the embed command.
+
+        Creates and sends an embed in the specified channel parsed from json.
+
+        Args:
+            ctx (Context): The context in which the command was called.
+            channel (str): The channel where to post the message. Can be channel name (starting with #) or channel id.
+            json_string (str): The json string representing the embed. Alternatively it could also be a pastebin link.
+        """
+        try:
+            if is_pastebin_link(json_string):
+                json_string = parse_pastebin_link(json_string)
+            embed_dict = json.loads(json_string)
+            embed = discord.Embed.from_dict(embed_dict)
+            await channel.send(embed=embed)
+        except commands.errors.CommandInvokeError:
+            await ctx.send("**__Error:__** Could not parse json. Make sure your last argument is valid JSON.")
+        except discord.errors.HTTPException:
+            await ctx.send("**__Error:__** Could not parse json. Make sure your last argument is valid JSON.")
+        except discord.DiscordException:
+            await ctx.send("**__Error:__** Invalid embed. Make sure you have at least title or description set (also "
+                           "for each additional field). You can validate your json at "
+                           "https://leovoel.github.io/embed-visualizer/.")
+        except ValueError as error:
+            await ctx.send(error)
+        except TypeError:
+            await ctx.send("**__Error:__** Error creating embed. Please check your parameters.")
+
+    @commands.group(name="bot", hidden=True)
+    @command_log
+    async def cmd_for_bot_stuff(self, ctx: commands.Context):
+        """Command handler for the `bot` command.
+
+        This is a command group regarding everything directly bot related. It provides a variety of subcommands for
+        special tasks like rebooting the bot or changing its Discord presence. For every single subcommand administrator
+        permissions are needed. If no subcommand has been provided, the corresponding help message will be posted
+        instead.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @cmd_for_bot_stuff.group(name="presence")
```

Comment:
Does this work, thought it had to have the group name not the function name

Reply:
Yes? Do you think I'm pushing my code without testing it beforehand? I feel offended. 😂
But no, it has to be the method name.